tensorflow_lite_support/python/test/task/vision/image_embedder_test.py

```diff
@@ -119,12 +120,9 @@ def test_embed(self, l2_normalize, quantize, with_bounding_box,
     image_feature_vector = image_result.embeddings[0].feature_vector
     self.assertLen(crop_result.embeddings, 1)
     crop_feature_vector = crop_result.embeddings[0].feature_vector
-    if quantize:
-      self.assertLen(image_feature_vector.value_string, 1024)
-      self.assertLen(crop_feature_vector.value_string, 1024)
-    else:
-      self.assertLen(image_feature_vector.value_float, 1024)
-      self.assertLen(crop_feature_vector.value_float, 1024)
+
+    self.assertLen(image_feature_vector.value, 1024)
```

Comment:
Please also assert the data type in image_feature_vector to check if the correct data type is returned depending on the quantize parameter value.

Reply:
@khanhlvg I've updated the tests to incorporate the change.